rwgame/DrawUI.cpp

```diff
@@ -35,15 +35,29 @@ constexpr float ui_mapSize = 150.f;
 constexpr float ui_worldSizeMin = 200.f;
 constexpr float ui_worldSizeMax = 300.f;
 
+static float hudScale = 1.f;
```

Comment:
You shouldn't use global variables which are not `const`. Better insert these into class.